hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusSubmitMessageHandler.java

```diff
@@ -507,22 +508,29 @@ public Fees calculateFees(@NonNull final FeeContext feeContext) {
         requireNonNull(feeContext);
         final var op = feeContext.body().consensusSubmitMessageOrThrow();
         final var calculatorFactory = feeContext.feeCalculatorFactory();
-        final var totalBytes = BASIC_ENTITY_ID_SIZE + op.message().length();
+        final var msgSize = op.message().length();
 
         final var topic =
                 feeContext.readableStore(ReadableTopicStore.class).getTopic(op.topicIDOrElse(TopicID.DEFAULT));
         if (topic != null && !topic.customFees().isEmpty()) {
             final var calculator = calculatorFactory.feeCalculator(SubType.SUBMIT_MESSAGE_WITH_CUSTOM_FEES);
             calculator.resetUsage();
+            // Charges the canonical price of $0.05 for a message up to 512 bytes and scales up to $0.06 for 1KB
```

Comment:
nit: non-blocking comment.
```suggestion
            // Charges the canonical price of $0.05 for a message up to 512 bytes and scales up to $0.06 for 'message.maxBytesAllowed' KB
```

Reply:
yes agreed Nana. Will fix in next PR to main. Thank you!